### ISSUE TITLE
"fade" animation not work properly in mega menu widget.

### ISSUE THREAD

**Issue by Michaelgimii** [elementor/elementor#22387]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When we select the "fade" animation for the mega menu content, this animation will only be executed during the display. When the mega menu content is closed, no animations are played.
Please see the mega menu on the [crocoblock website](https://crocoblock.com/changelog/). When the mega menu is displayed, it appears as a fade. Also, when the mega menu content is being hidden, it fades out. 

### Steps to reproduce

1- Select fade mode to display the content of the mega menu.

This animation only works when the mega menu content is displayed. This animation doesn't work when mega menu content is hidden, and mega menu content is quickly hidden.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

It cannot be useful.

**Comment by nicholaszein** [comment]
Hello @Michaelgimii! 

Thank you for submitting your issue! 🙏 

I can confirm this is happening, but I'm not sure this is a bug. I will check with the team. This might be simply a missing feature. Meaning this is not supported at the moment.

Kind regards

**Comment by Michaelgimii** [comment]
Hi @nicholaszein 

Yes, exactly. This is not a bug. But please consider it for future versions. It's really cool to be able to control the Entrance and exit animation separately in Mega Menu!

**Comment by cms-geek** [comment]
Hi @Michaelgimii,

have you possibly found a workaround?

Best regards,
Chris

**Comment by Michaelgimii** [comment]
Hi @Michaelgimii,

No, I hope Elementor adds it. It will definitely not be difficult for them. I also hope @nicholaszein will pass this on to the team.

**Comment by mateitudor** [comment]
It's definitely an unfinished component... they chose to go the "display: none;" route which means it's not going to be easy to animate smoothly from a visible state.

**Comment by lukepatrickillustrations** [comment]
Agree, Elementor needs to add a few features here. There should be an option for a fade out animation when leaving the menu all together. The fade animation should not occur when switching between multiple mega menus (so the mega menu does not close, just switches seamlessly). When exiting the mega menu or connected nav, there should be an animation delay option as well.

**Comment by Sergijus** [comment]
I've found a workaround with a simple CSS.
You need to force display: flex first, then utilize opacity to animate, and visibility to hide the main container.
You will just need to replace the #ID with the ID of your own container.

```
selector .e-n-menu-content, selector #e-n-menu-content-1442 {
    display: flex !important;
}

selector .e-n-menu-content {
    opacity: 0;
    visibility: hidden;
    transition: all 0.7s ease;
}

selector .e-n-menu-content.e-active {
    opacity: 1;
    visibility: visible;
    transition: all 0.7s ease;
}
```
